### DISCUSSION TITLE
Is it possible to get tenant domain in a recipe?

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#13735] [Q&A] [answered]
Good morning everyone, 

I have seen `[js: configuration(...)]` in the blog recipe. I would try to understand if exist a way to get tenant domain inside a recipe to configure a workflow without do manual modifications.

Thank you.

**Comment by jtkech** [comment, marked as answer]
You may need to implement a custom `IGlobalMethodProvider`, look at the classes implementing `IGlobalMethodProvider`, for example `UrlMethodsProvider`.

**Reply by PiemP** [reply]
Thank you @jtkech
